cmd/kops/edit.go

```diff
@@ -21,17 +21,29 @@ import (
 
 	"github.com/spf13/cobra"
 	"k8s.io/kops/cmd/kops/util"
+	"k8s.io/kubernetes/pkg/kubectl/cmd/templates"
+	"k8s.io/kubernetes/pkg/util/i18n"
 )
 
-func NewCmdEdit(f *util.Factory, out io.Writer) *cobra.Command {
-	cmd := &cobra.Command{
-		Use:   "edit",
-		Short: "Edit clusters and other resrouces.",
-		Long: `Edit a resource configuration.
+var (
+	edit_long = templates.LongDesc(i18n.T(`Edit a resource configuration.
 
-This command changes the cloud specification in the registry.
+    This command changes the cloud specification in the registry.
 
-It does not update the cloud resources, to apply the changes use "kops update cluster".`,
+    It does not update the cloud resources, to apply the changes use "kops update cluster".`))
+
+	edit_example = templates.Examples(i18n.T(`
+		# Edit a cluster configuration in AWS.
+		kops edit cluster k8s.cluster.site --state=s3://kops-state-1234
```

Comment:
So this is the root command, which I do not think edits, you have to use edit cluster or edit ig.